File: pm/src/main/java/org/sqlbuilder/pm/FnWordResolver.java
package org.sqlbuilder.pm;

import org.sqlbuilder.common.DeSerialize;
import org.sqlbuilder.common.Resolver;

import java.io.File;
import java.io.IOException;

public class FnWordResolver extends Resolver<String, Integer>
{
	public FnWordResolver(final String ser) throws IOException, ClassNotFoundException
	{
		super(DeSerialize.deserialize(new File(ser)));
	}
}

File: pm/src/main/java/org/sqlbuilder/pm/FnWordResolver.kt
package org.sqlbuilder.pm

import org.sqlbuilder.common.DeSerialize.deserialize
import org.sqlbuilder.common.Resolver
import java.io.File

typealias PmFnWordResolvable = String
typealias PmFnWordResolved = Int

class FnWordResolver(ser: String) : Resolver<PmFnWordResolvable, PmFnWordResolved>(deserialize<Map<PmFnWordResolvable, PmFnWordResolved>>(File(ser)))
